src/gsi/read_nsstbufr.f90

```diff
@@ -566,7 +566,6 @@ subroutine read_nsstbufr(nread,ndata,nodata,gstime,infile,obstype,lunout, &
            endif
 !
 !          Determine usage
-!
```

Comment:
Is this change intentional?

Reply:
> The deleting of this line was done "casually" though intentionally. I can add it back. BTW: the regression tests with the updated PR is still ongoing.

I'd add the `! ` back but it doesn't really matter.  

The branch for this PR, `TingLei-daprediction:feature/gsi_debug_omp` is now one commit behind `develop`.  

@ShunLiu-NOAA merged your PR #698 into `develop` this morning.  You can let the current regression tests run but you will need to rerun with the updated and rebuilt `develop` and `TingLei-daprediction:feature/gsi_debug_omp`.